src/bridge/cockpitrouter.c

```diff
@@ -709,11 +709,9 @@ static void
 process_kill (CockpitRouter *self,
               JsonObject *options)
 {
-  CockpitChannel *channel;
   GHashTableIter iter;
   const gchar *group = NULL;
   const gchar *host = NULL;
-  gpointer id, value;
   GList *list, *l;
 
   if (!cockpit_json_get_string (options, "group", NULL, &group))
```

Comment:
Do we want to allow killing the "default" group. I sort of think that we shouldn't but I don't have a hard opinion either way. Mainly wondering if you thought about it.